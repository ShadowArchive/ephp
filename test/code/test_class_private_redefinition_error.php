<?php

class A {
    public $a = 100;
}

class B extends A {
    private $a = 200;
}

