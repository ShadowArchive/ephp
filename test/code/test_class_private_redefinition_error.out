
Fatal error: Access level to B::$a must be public (as in class A) in {{CWD}}/test/code/test_class_private_redefinition_error.php on line 9
